src/services/expenses.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+let expenses = [];
+
+function getAllfilterExpenses(filters) {
+  let filteredExpenses = expenses;
+  const { userId, categories, from, to } = filters;
+
+  if (userId) {
+    filteredExpenses = filteredExpenses
+      .filter(expense => expense.userId === +userId);
```

Comment:
Either use the unary plus operator (+userId) or the Number constructor (Number(userId))
```suggestion
      .filter(expense => expense.userId === Number(userId));
```